src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -108,7 +108,7 @@ function ProctoredExamSettings({ courseId, intl }) {
 
   function handleSubmit(event) {
     event.preventDefault();
-    if (proctoringProvider === 'proctortrack' && !EmailValidator.validate(proctortrackEscalationEmail)) {
+    if (proctoringProvider === 'proctortrack' && !EmailValidator.validate(proctortrackEscalationEmail) && enableProctoredExams) {
```

Comment:
#NIT, can you check the `enableProctoredExams` condition first? That seems to be the most relevant check to be the first one.